### PR TITLE
Turn off low level task fusion

### PR DESCRIPTION
**Pull request by quasiben**
cc @madsbk

### CODE REVIEW

File: nightly-benchmark/run-dgx-benchmark.sh [hunk 1]
```diff
@@ -30,6 +30,7 @@ popd
 srun -N1 python nightly-run.py > dgx_raw_data.txt
 echo "Copy sitecustomize.py..."
 cp sitecustomize.py ${CONDA_PREFIX}/lib/python3.8/sitecustomize.py
+export DASK_DASK__OPTIMIZATION__FUSE__ACTIV=False
```
Comment (mrocklin) [inline nightly-benchmark/run-dgx-benchmark.sh]:
```suggestion
export DASK__OPTIMIZATION__FUSE__ACTIVE=False
```

Comment (quasiben) [reply nightly-benchmark/run-dgx-benchmark.sh]: Thanks!

**Comment by jakirkham** [comment]
Does Slurm inherit the environment variables from the current process when starting? Or do we need to pass a flag or something to it to have that effect?

**Comment by quasiben** [comment]
@jakirkham I believe so. The following is the output when running:

```bash
export DASK__OPTIMIZATION__FUSE__ACTIVE=False
srun -N1 python -c 'import dask; print(dask.config.config)'
```

```
/gpfs/fs1/bzaitlen/miniconda3/envs/20201204-nightly-0.17/bin/python
+ export DASK__OPTIMIZATION__FUSE__ACTIVE=False
+ DASK__OPTIMIZATION__FUSE__ACTIVE=False
+ srun -N1 python -c 'import dask; print(dask.config.config)'
{'_optimization': {'fuse': {'active': False}}, 'temporary-directory': None, 'dataframe': {'shuffle-compression': None}, 'array': {'svg': {'size': 120}, 'slicing': {'split-large-chunks': None}}, 'optimization': {'fuse': {'active': True, 'ave-width': 1, 'max-width': None, 'max-height': inf, 'max-depth-new-edges': None, 'subgraphs': None, 'rename-keys': True}}}
```

File: nightly-benchmark/run-dgx-benchmark.sh [hunk 2]
```diff
@@ -30,6 +30,7 @@ popd
 srun -N1 python nightly-run.py > dgx_raw_data.txt
 echo "Copy sitecustomize.py..."
 cp sitecustomize.py ${CONDA_PREFIX}/lib/python3.8/sitecustomize.py
+export DASK__OPTIMIZATION__FUSE__ACTIVE=False
```
Comment (jrbourbeau) [inline nightly-benchmark/run-dgx-benchmark.sh]: Looks like this is setting "_optimization.fuse.active"

```suggestion
export DASK_OPTIMIZATION__FUSE__ACTIVE=False
```

Comment (quasiben) [reply nightly-benchmark/run-dgx-benchmark.sh]: Thanks!  Good catch!

**Comment by jakirkham** [comment]
Thanks Ben! Also thanks everyone for the reviews! 😄

Anything else stick out? Or are we good to go?

**Comment by jakirkham** [comment]
Went ahead and merged. Though if anything else still needs to be addressed, please let us know and we can follow up. Thanks again for the pointer here! 😄

**Comment by mrocklin** [comment]
I'm eagerly awaiting the next automated issue :)

**Comment by jakirkham** [comment]
You and me both :)

**Pull request merged**